app.rb

```diff
@@ -0,0 +1,88 @@
+require './book'
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each { |book| puts "#{book.title} by #{book.author}" }
+  end
+
+  def list_people
+    @people.each { |person| puts "#{person.name} (#{person.class})" }
+  end
+
+  def create_person
+    puts 'Do you want to  create a student (1) or a teacher(2)?'
+    input_result = gets.chomp.to_i
+    print 'Name:'
+    name = gets.chomp
+    print 'Age:'
+    age = gets.chomp
+    case input_result
+    when 1
+      print 'Has Parent permission? [Y/N]'
+      permission = gets.chomp
+      student = Student.new(age, name, parent_permission: permission)
+      @people.push(student)
+      puts 'Student Created Successfully'
+    when 2
+      print 'Specialization:'
+      specialization = gets.chomp
+      teacher = Teacher.new(age, name, specialization:)
```

Comment:
- [ ] Still, there's an error thrown in an attempt to create a person of type teacher. Please ensure to improve this aspect.

**Error whiles creating a teacher person:**
![image](https://user-images.githubusercontent.com/12745474/227721731-998f5561-5183-4ed9-a50b-0e04d31083a5.png)